pkg/sdk/agent.go

```diff
@@ -90,28 +77,14 @@ func (sdk *agentSDK) Data(ctx context.Context, dataset agent.Dataset, privKey *r
 	ctx = metadata.NewOutgoingContext(ctx, md)
 	stream, err := sdk.client.Data(ctx)
 	if err != nil {
-		sdk.logger.Error("Failed to call Algo RPC")
+		sdk.logger.Error("Failed to call Data RPC")
 		return err
 	}
 	dataBuffer := bytes.NewBuffer(dataset.Dataset)
 
-	buf := make([]byte, bufferSize)
-	for {
-		n, err := dataBuffer.Read(buf)
-		if err == io.EOF {
-			break
-		}
-		if err != nil {
-			return err
-		}
-
-		err = stream.Send(&agent.DataRequest{Dataset: buf[:n]})
-		if err != nil {
-			return err
-		}
-	}
-
-	if _, err := stream.CloseAndRecv(); err != nil {
+	progressbar := progressbar.New()
```

Comment:
Do you need to create a new one, or you can reuse/reset old one if not used anymore?

Reply:
Yes, progress bar is used in Agent SDK so we need to create a new one every time the users uses the CLI to send data and algo.